vivo.owl

```diff
@@ -15,10 +15,12 @@
      xmlns:terms="http://purl.org/dc/terms/"
      xmlns:obo="http://purl.obolibrary.org/obo/"
      xmlns:statistics="http://purl.org/net/OCRe/statistics.owl#">
+     xmlns:doap="http://usefulinc.com/ns/doap#">	
 
     <owl:Ontology rdf:about="http://vivoweb.org/ontology/core">
         <rdfs:label xml:lang="en-US">VIVO Core Ontology</rdfs:label>
         <terms:license rdf:resource="https://spdx.org/licenses/Unlicense.html"/>
+	<doap:repository rdf:resource="https://github.com/vivo-ontologies/vivo-ontology"/>
```

Comment:
Not sure whether it is important, but officially Repository should be capitalized (http://usefulinc.com/ns/doap#). Moreover, there is also a more specific GitRepository class. Should we use that?
@hauschke do not hesitate to reject this suggestion if it is without sense
```suggestion
	<doap:GitRepository rdf:resource="https://github.com/vivo-ontologies/vivo-ontology"/>
```

Reply:
Hi @chenejac In this case the property <http://usefulinc.com/ns/doap#repository> is needed, not the class. It could be used in a different way, though, e.g. like this:

```
<owl:Ontology rdf:about="http://vivoweb.org/ontology/core">
    <doap:repository rdf:resource="https://github.com/vivo-ontologies/vivo-ontology"/> .


    <!-- http://usefulinc.com/ns/doap#Repository -->

    <owl:Class rdf:about="http://usefulinc.com/ns/doap#Repository">
        <rdfs:comment xml:lang="fr">Dépôt du code source.</rdfs:comment>
        <rdfs:comment xml:lang="de">Quellcode-Versionierungssystem.</rdfs:comment>
        <rdfs:comment xml:lang="es">Repositorio del código fuente.</rdfs:comment>
        <rdfs:comment xml:lang="pt">Repositório do código fonte.</rdfs:comment>
        <rdfs:comment xml:lang="en">Source code repository.</rdfs:comment>
        <rdfs:comment xml:lang="cs">Úložiště zdrojových kódů.</rdfs:comment>
        <rdfs:isDefinedBy rdf:resource="http://usefulinc.com/ns/doap#"/>
        <rdfs:label xml:lang="fr">Dépôt</rdfs:label>
        <rdfs:label xml:lang="es">Repositorio</rdfs:label>
        <rdfs:label xml:lang="de">Repository</rdfs:label>
        <rdfs:label xml:lang="en">Repository</rdfs:label>
        <rdfs:label xml:lang="pt">Repositório</rdfs:label>
        <rdfs:label xml:lang="cs">Úložiště</rdfs:label>
    </owl:Class>


    <!-- http://usefulinc.com/ns/doap#GitRepository -->

    <owl:Class rdf:about="http://usefulinc.com/ns/doap#GitRepository">
        <rdfs:subClassOf rdf:resource="http://usefulinc.com/ns/doap#Repository"/>
        <rdfs:comment xml:lang="fr">Dépôt Git du code source.</rdfs:comment>
        <rdfs:comment xml:lang="de">Git Quellcode-Versionierungssystem.</rdfs:comment>
        <rdfs:comment xml:lang="en">Git source code repository.</rdfs:comment>
        <rdfs:comment xml:lang="es">Repositorio Git del código fuente.</rdfs:comment>
        <rdfs:comment xml:lang="pt">Repositório Git do código fonte.</rdfs:comment>
        <rdfs:comment xml:lang="cs">Úložiště zdrojových kódů Git.</rdfs:comment>
        <rdfs:isDefinedBy rdf:resource="http://usefulinc.com/ns/doap#"/>
        <rdfs:label xml:lang="fr">Dépôt Git</rdfs:label>
        <rdfs:label xml:lang="de">Git Repository</rdfs:label>
        <rdfs:label xml:lang="en">Git Repository</rdfs:label>
        <rdfs:label xml:lang="es">Repositorio Git</rdfs:label>
        <rdfs:label xml:lang="pt">Repositório Git</rdfs:label>
        <rdfs:label xml:lang="cs">Úložiště Git</rdfs:label>
    </owl:Class>

    <!-- https://github.com/vivo-ontologies/vivo-ontology -->

    <owl:NamedIndividual rdf:about="https://github.com/vivo-ontologies/vivo-ontology">
        <rdf:type rdf:resource="http://usefulinc.com/ns/doap#GitRepository"/>
        <rdfs:seeAlso rdf:datatype="http://www.w3.org/2001/XMLSchema#anyURI">https://github.com/vivo-ontologies/vivo-ontology</rdfs:seeAlso>
    </owl:NamedIndividual>
```

Reply:
@chenejac, using just the property doap:repository instead of class seems to be less complicated.